it/v2/endpoints/UnignoreEmploymentControllerHipISpec.scala

```diff
@@ -0,0 +1,143 @@
+package v2.endpoints
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import common.errors.{EmploymentIdFormatError, RuleCustomEmploymentUnignoreError, RuleOutsideAmendmentWindowError}
+import common.support.EmploymentsIBaseSpec
+import play.api.http.HeaderNames.ACCEPT
+import play.api.http.Status._
+import play.api.libs.json.{JsObject, Json}
+import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.test.Helpers.AUTHORIZATION
+import shared.models.errors._
+import shared.services.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+
```

Comment:
Please make the same changes as mentioned in V1 apart from the removal of `RuleOutsideAmendmentWindowError`. No need to make the changes in the IFS one as it is going to be removed anyway, thanks.